wdl/tasks/NanoPlot.wdl

```diff
@@ -237,8 +237,8 @@ task NanoPlotFromBam {
         mem_gb:             32,
         disk_gb:            disk_size,
         boot_disk_gb:       10,
-        preemptible_tries:  0,
-        max_retries:        0,
+        preemptible_tries:  1,
```

Comment:
What's the purpose of this?
Nanoplot takes a long time to run typically, so we'd like to keep it this way to avoid blocking the whole pipeline.

Reply:
I went through and added these parameters to most of the tasks that MAS-ISO-seq uses.  I found that the `max_retries` was useful in the case where there was a spurious error that was not a preemption.  

I'll revert this for now though.